app/modules/munkiinfo/scripts/munkiinfo.py

```diff
@@ -96,6 +98,26 @@ def get_munkiprotocol():
     except AttributeError:
         return 'Could not obtain protocol'
 
+def middleware_checks():
+    """Check for middleware, get version if supported"""
+
+    middleware_version = []
+    name = []
+    for filename in os.listdir('/usr/local/munki'):
+            print "checking %s" % filename
+            if (filename.startswith('middleware') 
+                and os.path.splitext(filename)[1] == '.py'):
+                
+                name = os.path.splitext(filename)[0]
+                print 'middleware name %s' % name
+                try:
+                    middleware = importlib.import_module(name)
+                    middleware_version = middleware.get_version()
+                except:
+                    print "Error importing middleware for version checks, or version function does not exist."
```

Comment:
On second thought having this statement print when the verbose flag is passed via `managedsoftware` might be nice. I would add more info at the beginning of the statement 
```bash
print "ERROR: munkiinfo.py - Error importing middleware for version checks, or version function does not exist
```